src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/NuGet/ProjectAssetFileWatcherTests.cs

```diff
@@ -36,13 +36,12 @@ public class ProjectAssetFileWatcherTests
     foo.project.json, FilePath: ""C:\Foo\foo.project.json""", @"C:\Foo\foo.project.lock.json")]
         public async Task VerifyFileWatcherRegistration(string inputTree, string fileToWatch)
         {
-            var spMock = new IAsyncServiceProviderMoq();
```

Comment:
Looks like `IAsyncServiceProviderMoq` could be deleted now?

Reply:
Thanks will follow up.